assignments/callbacks.js

```diff
@@ -27,29 +27,61 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  return cb(arr.length)
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.length-1)
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y)
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x*y)
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if(list.includes(`${item}`) === true) {
+    cb(true)
+  } else {
+    cb(false)
+  }
 }
+  
+
+
+ 
 
 /* STRETCH PROBLEM */
 
 function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+   {
+    let object = {};
+    for (let i = 0; i < array.length; i++){
+      object[array[i]] = true;
+    }
+    cb(Object.keys(object));
+  }
+  let randomNum = [1, 5, 4, 7, 2, 4, 5, 10, 8];
+  
+  removeDuplicates(randomNum, function (noDuplicates){
+    console.log(noDuplicates)
+  });
 }
+
```

Comment:
Very cool, but it doesn't work without removing the curly braces that I suggested to above. 

Most importantly, do you understand what is going on here? It's ok if you didn't write it, but does it mostly make sense?